### ISSUE TITLE
Definition tooltip is not accessible to screen magnification users

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1477]
## Environment
Chrome 70
macOS Mojave

## Violation
A screen magnifier (zoom) user cannot read the definition tooltip as it relies on hover to be shown.
![nov-26-2018 11-24-27](https://user-images.githubusercontent.com/40970507/49030907-36082480-f16e-11e8-93a6-0e71197162e0.gif)

## Solution
A) The tooltip remains open as long as the users mouse remains _on top_ of the tooltip.
B) The tooltip is toggle-able open/close. This is the logical functionality given our use of a pointer on hover of the icon

## Reproduce
Using native macOS or Windows screen magnifying applications and a minimum 8x magnification.

**Comment by asudoh** [comment]
Thank you for reporting @dakahn! As we talked offline - Definition tooltip opens when the trigger text gets focus, and wondering if it works well for screen magnifier users.

**Comment by dakahn** [comment]
This didn't come up when testing with users with low-vision late last year. We'll hit it again during our spring audit.
